### PR TITLE
Assignment for week6

### PR DESCRIPTION
**Pull request by tungphamwd**

### CODE REVIEW

File: 03-task-manager/starter/middleware/error-handler.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { CustomAPIError } = require("../errors/custom-error");
+
+const errorHandlerMiddleware = (err, req, res, next) => {
+  if (err instanceof CustomAPIError) {
+    return res.status(err.statusCode).json({ msg: err.message });
+  }
+  return res
+    .status(500)
+    .json({ msg: "Something went wrong, please try again" });
```
Comment (joseluiz) [inline 03-task-manager/starter/middleware/error-handler.js]: Add the err variable to the JSON so the user can know what condition failed in the validation, for example, when the name is not provided to create the task.
You can do it like this:
return res.status(500).json({ msg: "Something went wrong, try again later", err});

**Review by joseluiz — CHANGES_REQUESTED** [review]
Hi Tung. Your code is everything correct, I left only a suggestion for improving the error handler. You forgot to complete the additional assignment, create the QuizAnswers2.txt, and answer the questions.

File: 03-task-manager/starter/middleware/error-handler.js (hunk 1 above)
Comment (tungphamwd) [reply 03-task-manager/starter/middleware/error-handler.js]: Thanks @joseluiz for your kindly feedback. I already changed the code. Please check it. I appreciate it.

**Review by joseluiz — APPROVED** [review]
Hey @tungphamwd. It looks good now. I'm approving your PR. Good job!

**Pull request merged**